packages/protocol/contracts/L1/hooks/AssignmentHook.sol

```diff
@@ -63,6 +64,7 @@ contract AssignmentHook is EssentialContract, IHook {
     function onBlockProposed(
         TaikoData.Block memory blk,
         TaikoData.BlockMetadata memory meta,
+        bytes32 parentMetaHash,
```

Comment:
If you want to keep the function parameters smaller, the value of `parentMetaHash` can be loaded from `meta.parentMetaHash`.